interpreter/cling/lib/Interpreter/CIFactory.cpp

```diff
@@ -1362,6 +1362,11 @@ namespace {
       argvCompile.push_back("-fno-omit-frame-pointer");
 #endif
 
+#ifdef __cpp_sized_deallocation
```

Comment:
@pcanal, was trying to debug this yesterday. Maybe he has an opinion. I don't because that feature becomes default in llvm19.

Reply:
The patch comes from him actually.

Reply:
Then this lgtm!

Reply:
Although we do not really understand the underlying issue...